provider/ansible.py

```diff
@@ -149,6 +150,18 @@ def distro_install():
         """
         Install ansible from the distro
         """
+        # Provide custom dnf repo containing ansible
+        if "ansible_repo" in params and params["ansible_repo"] != "":
+            repo_options = {
+                "priority": "1",
+                "gpgcheck": "0",
+                "skip_if_unavailable": "1"
+            }
+            yum_backend = YumBackend()
+            if yum_backend.add_repo(params["ansible_repo"], repo_options):
+                LOG_JOB.info(f"Ansible repo was added: {params['ansible_repo']}")
+            else:
+                LOG_JOB.error("Ansible repo was required, but failed to be added.")
```

Comment:
Suggest removing repo from testing system after pkg installed.

Reply:
Yes, I agree. It will be safer approach. (Added.)